extensions/amp-accordion/1.0/storybook/Basic.amp.js

```diff
@@ -56,18 +56,63 @@ export const _default = () => {
       </amp-accordion>
 
       <div class="buttons" style={{marginTop: 8}}>
-        <button on={createApiString('toggle', 'section1')}>
+        <button on={createApiString('tap', 'toggle', 'section1')}>
```

Comment:
From the test point of view, this is pretty hard to read. As much as possible it'd be nice if these Storybooks "reminded" us of the AMP documents. IMHO the repetition is annoying, but one of few cases where it's better.

Reply:
I totally agree that this is harder to read.  Unfortunately, amp does not like a few forbidden strings like `.expand` and `.collapse`.  So this was my workaround

Reply:
Can we un-forbid the key words in `storybook/` paths?

Reply:
We 100% should. We just need to skip storybook files somewhere [here](https://github.com/ampproject/amphtml/blob/9b31ca7f5b39051207e33e3d5cd856353de62ff3/build-system/tasks/presubmit-checks.js#L1344).

Reply:
Awesome, excluded all files with pathname ending in `storybook/*something.js` and updated storybook.

https://github.com/ampproject/amphtml/pull/30976/commits/67fb1a99890e63d16285fec78374609fb71781be